### PR TITLE
Update launch.json to work with the new launcher

### PR DESCRIPTION
**Pull request by dragunoff**
Tested only on linux as I don't have any other OS at the moment.

### CODE REVIEW

**Review by Mailaender — DISMISSED** [review]
I can confirm that this fixes the problem. Thanks.

File: .vscode/launch.json [hunk 1]
```diff
@@ -11,9 +11,9 @@
 				"type": "mono"
 			},
 			"request": "launch",
-			"program": "${workspaceRoot}/OpenRA.Game.exe",
-			"cwd": "${workspaceRoot}",
-			"args": ["Game.Mod=cnc"],
+			"program": "${workspaceRoot}/bin/OpenRA.exe",
+			"cwd": "${workspaceRoot}/bin",
+			"args": ["Game.Mod=cnc", "Engine.EngineDir='..'"],
```
Comment (abcdefg30) [inline .vscode/launch.json]: Changing this to just `"Engine.EngineDir=.."` fixes the issue for me.

**Review by abcdefg30 — COMMENTED** [review]
This throws `Exception of type 'System.IO.DirectoryNotFoundException': C:\OpenRA\bin\'..'` so it looks like it somehow concatenates the path wrongly?

**Comment by teinarss** [comment]
This will needed to be update again after we merge the net 5 PR.

**Comment by pchote** [comment]
We'll still want the non-net5 version to pick to the prep branch.

File: .vscode/launch.json (hunk 1 above)
Comment (phrohdoh) [reply .vscode/launch.json]: You certainly need the `program` changes, too.
The `cwd` change should be dropped IMO as that doesn't match what the `launch-game` script does and is unnecessary.

**Review by phrohdoh — DISMISSED** [review]
LGTM and confirmed I can build & launch again via VSCode on macOS (Big Sur, 11.1).

**Comment by dragunoff** [comment]
Update: Rebased on latest bleed, changed args to "Engine.EngineDir=.." and removed the `cwd` line. Tested on linux.

**Comment by phrohdoh** [comment]
Reconfirming this (still) works on macOS. 👍

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/6d1358f9c787d9651da77efb2c983081cc3f2c78...cf3c4dd5d3afaf5f8bf9e988c41170922a9e70d1)

https://github.com/OpenRA/OpenRA/commit/9c4dd4a08a89d7766efb0a2315fba280ba95c78c

**Pull request merged**